src/index.js

```diff
@@ -92,6 +103,10 @@ function unregisterKeyCommands(keyCommands) {
  * @returns {Object} eventListener instance to register a callback.
  */
 function getEventEmitter() {
+    if (Platform.OS === 'web') {
+        return KeyCommand.EventEmitter;
+    }
```

Comment:
@luacmartins Do we have to go with platform-specific files for this repo as well?

Reply:
Yes, let's keep it consistent with the practices we have in App

Reply:
@azimgd let's update it.

Reply:
+1

Reply:
updated